### PR TITLE
Add Admonitions to Learn >> Adv + Keys Content

### PR DESCRIPTION
**Pull request by salmad3**
Part of https://github.com/w3f/polkadot-wiki/issues/3133

### CODE REVIEW

File: docs/learn/learn-availability.md [hunk 1]
```diff
@@ -24,8 +24,12 @@ There are five phases of the Availability and the Validity protocol.
 The parachain phase of AnV is when the _collator_ of a parachain proposes a _candidate block_ to the
 validators that are currently assigned to the parachain.
 
-> A **candidate block** is a new block from a parachain collator that may or may not be valid and
-> must go through validity checks before being included into the Relay Chain.
+:::note Candidate block
+
+A new block from a parachain collator that may or may not be valid and
```
Comment (profradha) [inline docs/learn/learn-availability.md]: Title is apt. The line reads better in full.
```suggestion
A **candidate block** is a new block from a parachain collator that may or may not be valid and
```

File: docs/learn/learn-cross-consensus.md [hunk 2]
```diff
@@ -29,9 +29,13 @@ with the following properties:
     It is _extensible_ and, in turn, _future-proof_ and _forwards-compatible_.
 - Highly efficient to operate in a tightly constrained and metered environment, as is the case with many chains.
 
-> XCM is not designed in that every system supporting the format is expected to be able to interpret any
-> possible XCM message. Practically speaking, one can imagine that some messages will not have reasonable
-> interpretations under some systems or will be intentionally unsupported.
+:::info XCM design
```
Comment (profradha) [inline docs/learn/learn-cross-consensus.md]: No title needed for this note
```suggestion
:::info
```

File: docs/learn/learn-keys.md [hunk 3]
```diff
@@ -141,13 +145,15 @@ Not yet, but they will be. BLS signatures allow more efficient signature aggrega
 GRANDPA validators are usually signing the same thing (e.g. a block), it makes sense to aggregate
 them, which can allow for other protocol optimizations.
 
-As stated in the BLS library's README,
+:::info As stated in the BLS library's README
```
Comment (profradha) [inline docs/learn/learn-keys.md]:
```suggestion
:::info From the BLS library's README
```

File: docs/learn/learn-phragmen.md [hunk 4]
```diff
@@ -16,11 +16,15 @@ future.
 The quote below taken from the reference [Phragmén paper](#external-resources) sums up the purpose
 of the sequential Phragmén method:
 
-> The problem that Phragmén’s methods try to solve is that of electing a set of a given numbers of
-> persons from a larger set of candidates. Phragmén discussed this in the context of a parliamentary
-> election in a multi-member constituency; the same problem can, of course, also occur in local
-> elections, but also in many other situations such as electing a board or a committee in an
-> organization.
+:::info Electing a set of given numbers from a larger set
```
Comment (profradha) [inline docs/learn/learn-phragmen.md]:
```suggestion
:::info 
```

File: docs/learn/learn-proxies.md [hunk 5]
```diff
@@ -141,20 +157,27 @@ next to it). The `addProxy(proxy, proxy_type)` function will need to be selected
...
 
 ### Creating Anonymous Proxies on Polkadot-JS UI
 
-> NOTE: The first anonymous proxy you add should always be of *type* `Any`.
-> Also, if there are multiple anonymous proxies for the proxied account, you should keep at least  *one* `Any`
-> type.
-> Without having an `Any` type proxy, you won't be able to send funds, add new proxies, kill the
-> anonymous proxy or take any action not specifically allowed by the types of the proxies the
-> account has.
+:::caution Anonymous proxy types
```
Comment (profradha) [inline docs/learn/learn-proxies.md]: This caution is about setting at least one anonymous proxy of **any** type. Anonymous proxy types title is misleading. The caution should be okay without a title
```suggestion
:::caution 
```

File: docs/learn/learn-proxies.md [hunk 6]
```diff
@@ -179,19 +202,25 @@ where you will be able to select the type of proxy for that specific account.
 
 ![proxy generation part 2](../assets/polkadot_add_another_proxy.png)
 
-> NOTE: You cannot create an anonymous function from the Accounts page, you must be on the
-> Extrinsics page.
+:::info Creating an anonymous function
```
Comment (profradha) [inline docs/learn/learn-proxies.md]:
```suggestion
:::info 
```

File: docs/learn/learn-proxies.md [hunk 7]
```diff
@@ -179,19 +202,25 @@ where you will be able to select the type of proxy for that specific account.
...
+
 If you want to remove a proxy, there are a few functions on the extrinsic page that will help do
 this.
 
 For non-anonymous proxies, you can use `removeProxy` or `removeProxies`, but must use the `killAnonymous` function for anonymous proxies. This must be called **from** the *anonymous* proxy. This means that the anonymous proxy must be added as an account to Polkadot-JS accounts.
 
-The following steps can be used to remove your proxy:
+:::
 
-> WARNING: there is no way to get access to the proxy after deleting it.
```
Comment (profradha) [inline docs/learn/learn-proxies.md:194]: This warning applies only to anonymous proxy. Not every proxy.

File: docs/learn/learn-proxies.md [hunk 8]
```diff
@@ -179,19 +202,25 @@ where you will be able to select the type of proxy for that specific account.
...
+:::info Creating an anonymous function 
+
+You cannot create an anonymous function from the Accounts page, you must be on the
+Extrinsics page.
+
+:::
 
 ### Removing Proxies
 
+:::warning There is no way to get access to the proxy after deleting it
```
Comment (profradha) [inline docs/learn/learn-proxies.md]: Not true. It applies only for anonymous proxy. You can remove a staking proxy and still have access to that staking proxy account.

**Review by DrW3RK — CHANGES_REQUESTED** [review]

File: docs/learn/learn-phragmen.md (hunk 4 above)
Comment (salmad3) [reply docs/learn/learn-phragmen.md]: changed to a note

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**